tool/release/createnotes/CreateNotes.kt

```diff
@@ -21,27 +21,36 @@
 
 package com.vaticle.dependencies.tool.release.createnotes
 
+import java.nio.file.Files
+import java.nio.file.Path
 import java.nio.file.Paths
 import kotlin.io.path.notExists
 
 fun main(args: Array<String>) {
     val bazelWorkspaceDir = Paths.get(getEnv("BUILD_WORKSPACE_DIRECTORY"))
     val githubToken = getEnv("CREATE_NOTES_TOKEN")
-    if (args.size != 5) throw RuntimeException("org, repo, version, commit, and template must be supplied")
-    val (org, repo, version, commit, templateFileLocation) = args
+    if (args.size != 5) throw RuntimeException("org, repo, commit, version, and template must be supplied")
+    val (org, repo, commit, version, templateFileLocation) = args
     val templateFile = bazelWorkspaceDir.resolve(templateFileLocation)
     if (templateFile.notExists()) throw RuntimeException("Template file '$templateFile' does not exist.")
 
-    println("Repository: $org/$repo")
-    println("Commit: $commit")
+    println("Commit: $org/$repo@$commit")
     println("Version: $version")
 
-    val commits = getCommits(org, repo, Version.parse(version), commit, bazelWorkspaceDir, githubToken)
-    println("There are ${commits.size} commits to be collected.")
-    val commitDescriptions = getCommitDescriptions(org, repo, commits, githubToken)
-    writeReleaseNoteMd(commitDescriptions, templateFile)
+    val commits = collectCommits(org, repo, commit, Version.parse(version), bazelWorkspaceDir, githubToken)
+    println("Found ${commits.size} commits to be collected into the release note.")
+    val notes = collectNotes(org, repo, commits, githubToken)
+    writeNotesMd(notes, templateFile)
 }
 
 private fun getEnv(env: String): String {
     return System.getenv(env) ?: throw RuntimeException("'$env' environment variable must be set.")
-}
\ No newline at end of file
+}
+
+private fun writeNotesMd(notes: List<Note>, releaseTemplateFile: Path) {
+    val template = String(Files.readAllBytes(releaseTemplateFile), Charsets.UTF_8)
```

Comment:
@lolski consider using [`readText`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.io/java.io.-file/read-text.html#readtext) instead (`Charsets.UTF_8` is the default so can be omitted):

```suggestion
    val template = releaseTemplateFile.toFile().readText()
```

Reply:
Fixed in 3be6d1dd53f720c2490e05d2d4d2e77b078fdac1.